src/client/sky.h

```diff
@@ -145,4 +146,13 @@ class Sky : public scene::ISceneNode
 	video::ITexture *m_moon_texture;
 	video::ITexture *m_sun_tonemap;
 	video::ITexture *m_moon_tonemap;
+	void draw_sun(video::IVideoDriver *driver, float sunsize, video::SColor suncolor,
+			video::SColor suncolor2, float wicked_time_of_day);
+	void draw_moon(video::IVideoDriver *driver, float moonsize, video::SColor mooncolor,
+			video::SColor mooncolor2, float wicked_time_of_day);
+	std::array<video::S3DVertex, 4> draw_sky_body(
+					float pos_1, float pos_2, video::SColor c);
+	std::array<video::S3DVertex, 4> place_sky_body(
+					std::array<video::S3DVertex, 4> vertices, float horizon_position,
+					float day_position);
```

Comment:
For the 2 'std::array' functions please indent by 2 tabs and in the same way as 'draw sun/moon'.